GNUmakefile

```diff
@@ -44,8 +44,13 @@ test: fmtcheck
     		xargs -t -n4 go test -tags "all" $(TESTARGS) -timeout=30s -parallel=4
 
 testacc: fmtcheck
+	@echo "==> Sourcing .env file if avaliable"
+	if [ -f .env ]; then set -o allexport; source .env; set +o allexport; fi
```

Comment:
Is this `.env` file bind with vscode? If it is, don't recommend add this config file here.  `make` should not  tied to any specific specify development tool configuration.

Reply:
No, it's just a .env file that you can set all of your config in and if it's there will be picked up by the tooling. Not VSCode specific.

Reply:
I'll be merging this PR, because it's gotten a bit out of hand, if this is still an issue I can make a new PR to address this :)